backend/core/serializers.py

```diff
@@ -130,3 +129,19 @@ class Meta:
         fields = ["message"]
 
     message = serializers.CharField()
+
+
+class BulkSetDueTimeSerializer(serializers.Serializer):
+    class Meta:
+        fields = [
+            "due_time"
+            "flavours",
+            "content_item",
+            "team"
```

Comment:
since the team comes from the url, why does the serializer also need a team field?

Reply:
These were the required fields on the issue.